.github/workflows/pr-labeler.yml

```diff
@@ -31,7 +31,7 @@ jobs:
     # - "config/crd/bases/.*"  # generated CRDs
     # - "docs/apis/v*.md"      # generated API documentation,
     #                          # ex. docs/apis/v1alpha1.md
-    - uses: srvaroa/labeler@v0.9
+    - uses: srvaroa/labeler@master
```

Comment:
We want to avoid sliding version targets. Please point to a specific version instead.

Reply:
Thanks for the review, fixed!